ZFitter/Makefile

```diff
@@ -3,6 +3,12 @@ include MakeVars
 #### Make the list of modules from the list of .cc files in the SRC directory
 MODULES=$(shell ls $(SRCDIR)/*.cc | sed "s|.cc|.o|;s|$(SRCDIR)|$(OBJ_DIR)|g")
 MODULES_EXT=-L$(CMSSW_BASE)/lib/$(SCRAM_ARCH)/ -L$(CMSSW_RELEASE_BASE)/lib/$(SCRAM_ARCH)/
+
+-include $(CMSSW_BASE)/.SCRAM/$(SCRAM_ARCH)/MakeData/Tools/cmssw.mk
```

Comment:
This file only exists for CMSSW Versions with patchN
but it has a variable that points to the library of the base CMSSW version